sdh-proxy.c

```diff
@@ -65,9 +65,18 @@ int do_exit = 0;
 // Toggled with -d on command line. Displays debug info. 
 int debug = 0;
 
+// Toggled with -l on command line. Write Stats to stat.log
+int logstat = 0;
+char logstatfile[] = "sdh.stat";
+
 // Does nothing at the moment. Will enable rewriting subnet info if relevant. 
 int do_network_rewrite = 0; // Rewrite IP broadcast address for the new
                             // network interface
+int32_t pkt_rx = 0;
+int32_t pkt_tx = 0;
+int32_t pkt_drop = 0;
+unsigned short int pkt_stats[65535];
```

Comment:
Ditto for this - I expect the short would overflow pretty regularly? Maybe in the future we should make this a hash table so we don't need a giant array of mostly 0s, or somehow index the array based on the ports listed in the ports file. Not important though, just thinking out loud